trestle/core/commands/author/profile.py

```diff
@@ -303,7 +303,7 @@ def assemble_profile(
         """
         md_dir = trestle_root / md_name
         profile_path = trestle_root / f'profiles/{orig_profile_name}/profile.json'
-        _, _, orig_profile = ModelUtils.load_distributed(profile_path, trestle_root, prof.Profile)
```

Comment:
Please add this as the test case to ensure the exception is raised

Reply:
added test for load_dist with Dict collection class.